lib/Storage/AmazonS3.php

```diff
@@ -311,7 +312,7 @@ public function stat($path) {
 				]);
 
 				$stat['size'] = $result['ContentLength'] ?: 0;
-				if (isset($result['Metadata']['lastmodified'])) {
+				if ((int)$result['Metadata']['lastmodified'] !== null) {
```

Comment:
```
<?php
$x['top']['sub'] = "something";
$y = (int)$x['top']['lastmodified'];
var_dump($y);```
```
NOTICE Undefined index: lastmodified on line number 3
int(0) 

What was the problem with `isset()` ?

Maybe:
```suggestion
				if (isset($result['Metadata']['lastmodified']) && ($result['Metadata']['lastmodified'] !== null)) {
```

Reply:
phan was too fussy about `$result` having 2 levels of strings keys in the array. I added some comments and suppressed the phan error.